### PR TITLE
updated upgrades page with safe upgrade feature

### PR DESCRIPTION
**Pull request by richbibby**

### CODE REVIEW

File: docs/Administration Console/upgrading-nbc.md [hunk 1]
```diff
@@ -1,35 +1,53 @@
...
 
-4. The upgrade will take a few minutes to complete. you can monitor the progress of the upgrade by navigating down to the **Logs** section, and clicking the refresh icon on the right hand side:
+4. The **Safe Upgrade tool** will present you with the **potential** and **recommended** versions you can upgrade to. You will see the version number, the status, and the whether or not installed plugins are compatible with the target NetBox version, plus a link to the release notes for that version.  
 
-    ![refresh logs](../images/console/refesh_logs.png)
+    ![manage instance](..//images/upgrades/upgrades_upgrade_options_3.png)
 
-5. Then click on the name of the instance with the latest timestamp (you may see the state change from `pending` to `running`). This will display the output from the log of your upgraded instance as it starts up:
+    Note that in this case the **NetBox BGP** and **NetBox Topology Views** plugins are at a version that is compatible with NetBox version 3.5.9. All Versions higher than this (up to 3.6.6) will require an upgrade to the plugins also, but don't worry the safe upgrade tool will handle all of that for you.
```
Comment (rboucher-me) [inline docs/Administration Console/upgrading-nbc.md:23]: a nit, but should "Versions" be lower case?

File: docs/Administration Console/upgrading-nbc.md [hunk 2]
```diff
@@ -1,35 +1,53 @@
...
 
-5. Then click on the name of the instance with the latest timestamp (you may see the state change from `pending` to `running`). This will display the output from the log of your upgraded instance as it starts up:
+    Note that in this case the **NetBox BGP** and **NetBox Topology Views** plugins are at a version that is compatible with NetBox version 3.5.9. All Versions higher than this (up to 3.6.6) will require an upgrade to the plugins also, but don't worry the safe upgrade tool will handle all of that for you. 
 
-    ![view log file](../images/console/view_logs.png)
+    Also note that we could upgrade to any of the other **potential** options, but that the Safe Upgrade tool is recommending an upgrade to version 3.6.6, and if we select this option, both the plugins will be automatically upgraded to compatible versions as part of the upgrade process.
 
-5. Refresh the logs again, and after a few minutes you should soon see only one running instance (or two if you have a HA deployment):
+    Note too that if a plugin is **not compatible** with a certain netbox version then you will see a warning like this:
```
Comment (rboucher-me) [inline docs/Administration Console/upgrading-nbc.md:27]: netbox or NetBox?

**Review by rboucher-me — APPROVED** [review]

File: docs/Administration Console/upgrading-nbc.md (hunk 1 above)
Comment (richbibby) [reply docs/Administration Console/upgrading-nbc.md:23]: yes, will fix

File: docs/Administration Console/upgrading-nbc.md (hunk 2 above)
Comment (richbibby) [reply docs/Administration Console/upgrading-nbc.md:27]: will fix

**Pull request merged**